config/logstash.yml

```diff
@@ -165,6 +165,15 @@
 # api.auth.basic.username: "logstash-user"
 # api.auth.basic.password: "s3cUreP4$$w0rD"
 #
+#
+# Applies when `api.auth.type` `basic`.
+# Sets Logstash to WARN or ERROR when HTTP authentication password doesn't meet the password policy requirements.
+# The default is WARN. Setting to ERROR enforces stronger passwords (recommended).
+# The default password policy requires non-empty minimum 5 char string that includes a digit, upper case letter and lower case letter.
+# Set `api.auth.basic.password_policy.include.symbol` `REQUIRED` to accept only passwords that include at least one special character.
+# Set `api.auth.basic.password_policy.length.minimum` to a value from 6 to 1024 if you want to require more than the 5 character default setting for passwords.
```

Comment:
```suggestion
# Set `api.auth.basic.password_policy.length.minimum` to 6 or higher if you want to require more than the 5 character default setting for passwords..
```

Reply:
I understand that we need an upper limit programmatically. But because it's high enough to defy practical application, I don't think we should expose that value in the documentation. 

If users are legit reading the docs to get their work done, this approach ("6 or higher") will be appreciated.  If somebody tries to set a value beyond 1024 to see what will happen, then they'll see an error, right?

Reply:
Yup, they will face `Password must be length of between {minUserSetValue or 5} and 1024` error or warn depending on policy mode set.
I also think we don't need to expose maximum length. Robust password generators also limit this to 128 but when I did some research, extra robust systems uses 1024.
How do you think if we keep it in docs (`settings-file.asciidoc`) but remove from `logstash.yml` config explanation?  So that if customers want details they will be able to figure out.